plat/arm/board/fvp/include/plat_macros.S

```diff
@@ -42,7 +42,7 @@
 	 * Clobbers: x0 - x10, x16, x17, sp
 	 * ---------------------------------------------
 	 */
-	.macro plat_print_gic_regs
+	.macro plat_crash_print_regs
```

Comment:
`plat_print_interconnect_regs()' in cci_macros.S is no longer called for this platform. That function should be renamed to `print_cci_regs()` and explicitly called at the end of this function. Also the comment here will need updating to say both GIC and CCI regs are printed.